deltachat-ffi/deltachat.h

```diff
@@ -528,9 +528,20 @@ int             dc_is_io_running(const dc_context_t* context);
 void            dc_stop_io(dc_context_t* context);
 
 /**
- * This function can be called whenever there is a hint
- * that the network is available again.
- * The library will try to send pending messages out.
+ * This function should be called when there is a hint
+ * that the network is available again,
+ * eg. as a response to system event reporting network availability.
+ * The library will try to send pending messages out immediately.
+ *
+ * Moreover, to have a reliable state
+ * when the app comes to foreground with network available,
+ * it may be reasonable to call the function also at that moment.
+ *
+ * It is okay to call the function unconditionally when there is
+ * network available, however, calling the function
+ * _without_ having network may interfere with the backoff algorithm
+ * and will led to let the jobs fail faster, with fewer retries
+ * and may avoid messages being sent out.
```

Comment:
That's true if I understood the code correctly but maybe we should rather change this behavior?
Like, https://github.com/deltachat/deltachat-core-rust/pull/1621

Reply:
this pr is about commenting the current state.
we can change this documentation in #1621 then.

Reply:
k, #1621 is not that clear, imho